projects/ledger-app/0001-token-admin-methods.md

```diff
@@ -0,0 +1,219 @@
+# Summary
+[summary]: #summary
+
+Support for admin operations for custom tokens on the Ledger app.
+This includes:
+- Creating tokens and NFTs.
+- Minting new tokens and NFTs.
+- Melting new tokens and NFTs.
+- Delegating authorities.
+- Destroying authorities.
+
+# Motivation
+[motivation]: #motivation
+
+Today Hathor's Ledger app only allow sending transactions with custom tokens, so users and use-cases can hold and send custom tokens but to create and manage token supply it is required to use a software wallet.
+This design is to allow users who use Ledger, a hardware cold wallet, to fully manage custom tokens without using any software wallet.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+> [!WARNING] Request for comment
+> This whole paragraph below can change if we really want to, we just need to add an `isNFT` boolean in the token data for the user to confirm, so in all transactions the Ledger will be informed that the token is an NFT with the user express consent during token signing

```

Comment:
I would use a `number of decimals` parameter (instead of `isNFT`). In this case, NFTs would use `decimals=0`.

Reply:
Adding this new parameter will be addressed in another design since it requires some discussion on UI/UX (e.g. if we should add scrolling when using a lot of decimals).